### PR TITLE
Fix io.anserini.ltr.FeatureExtractorCli for argument -collection clueweb

### LINKED ISSUE #688: FeatureExtractorCli fails on collection clueweb
If I pass `-collection clueweb` to `io.anserini.ltr.FeatureExtractorCli` it fails since the condition in line 99 checks for a collection named 'webxml '.

Later, the `WebxmlTopicReader ` package path has changed.

### PR DESCRIPTION
**Pull request by mam10eks**
This pull request will close #688.

I would be happy if I could improve the robustness of `io.anserini.ltr.FeatureExtractorCli` further by adding some integration-tests.
If there is some interest in that just let me know, I would be glad to help.

### CODE REVIEW

File: src/main/java/io/anserini/ltr/FeatureExtractorCli.java [hunk 1]
```diff
@@ -64,15 +65,40 @@
     @Option(name = "-extractors", metaVar = "[path]", required = false, usage = "FeatureExtractors File")
     public String extractors = null;
 
+    @SuppressWarnings("unchecked")
+	public <K> TopicReader<K> buildTopicReaderForCollection() throws Exception {
+		if ("clueweb".equals(collection)) {
+			return new WebxmlTopicReader(Paths.get(topicsFile));
+		}
+		else if ("gov2".equals(collection)){
```
Comment (lintool) [inline src/main/java/io/anserini/ltr/FeatureExtractorCli.java]: we use two space indent, and else with preceding braces.

**Comment by lintool** [comment]
Thanks for your contribution!

Other than minor formatting changes, we can merge.

**Comment by lintool** [comment]
@mam10eks do we have an update on how this is going?

**Comment by mam10eks** [comment]
Hi @lintool,

I am still working on this.
I have found some related problems in the RankLibReranker and this motivated me to clean up both parts.

Since I am working on this in my spare time I think I need two more weeks until I can finish this and have everything properly tested.

My goal is to reproduce some results reported in the "Learning to Rank for Information Retrieval" book (Tie-Yan Liu) with the Anserini-LTR-Pipeline to be sure that everything is fine.

**Comment by MathBunny** [comment]
@mam10eks How is the progress going? Are you still working on it?

**Comment by mam10eks** [comment]
Hi @MathBunny,
sorry for the long delay.

I had planned too much and there have been some changes in the Master Branch since then.
I will fix those stuff tomorrow and then provide a small merge request instead of a large one.

**Comment by mam10eks** [comment]
@MathBunny,

From my side this can be merged now.

There are still other issues in the LTR-Pipeline.
Is anyone else working on that?
I still work on that in my spare time, but it needs some weeks (I will split this into "smaller merge requests")
.

**Comment by lintool** [comment]
hi @MathBunny can you take a look? you're more familiar with the code than I am at this point... do a CR and if it's okay we'll merge.

**Pull request merged**